port/common/omrfile.c

```diff
@@ -607,41 +607,33 @@ EsTranslateOpenFlags(int32_t flags)
 void
 omrfile_vprintf(struct OMRPortLibrary *portLibrary, intptr_t fd, const char *format, va_list args)
 {
-	char outputBuffer[256];
-	char *allocatedBuffer;
-	uint32_t numberWritten;
-	va_list copyOfArgs;
-
-	/* Attempt to write output to stack buffer */
-	COPY_VA_LIST(copyOfArgs, args);
```

Comment:
You are correct it takes a va_list but as you consume args from a va_list it moves the list along and you can not get back to the beginning again.  If there are multiple calls using the same va_list I believe you will not see any values in the second, third, etc usage of the same va_list.  I believe that is why the tests were failing originally with your changes.